changes/changes-2.1.0.md

```diff
@@ -0,0 +1,9 @@
+Changes by Version
+==================
+Release Notes.
+
+Apollo 2.1.0
+------------------
```

Comment:
Please update the [CHANGES.md](https://github.com/apolloconfig/apollo/blob/master/CHANGES.md).